project.yml

```
@@ -27,6 +27,27 @@ packages:
   SwiftPrettyPrint:
     url: https://github.com/YusukeHosonuma/SwiftPrettyPrint
     version: 1.0.0
+  FirebaseAnalytics:
+    url: https://github.com/firebase/firebase-ios-sdk.git
+    version: 7.3.1
+  FirebaseAppDistribution-Beta:
+    url: https://github.com/firebase/firebase-ios-sdk.git
+    version: 7.3.1
+  FirebaseCrashlytics:
+    url: https://github.com/firebase/firebase-ios-sdk.git
+    version: 7.3.1
+  FirebaseFirestore:
+    url: https://github.com/firebase/firebase-ios-sdk.git
+    version: 7.3.1
+  FirebaseInstallations:
+    url: https://github.com/firebase/firebase-ios-sdk.git
+    version: 7.3.1
+  FirebaseMessaging:
+    url: https://github.com/firebase/firebase-ios-sdk.git
+    version: 7.3.1
+  FirebaseRemoteConfig:
+    url: https://github.com/firebase/firebase-ios-sdk.git
+    version: 7.3.1
```

`url` と `version` をまとめる方法があれば知りたいです 🙏 
同じのを何回も書くのは冗長なので…

---

```suggestion
  Firebase:
    url: https://github.com/firebase/firebase-ios-sdk
    from: 7.3.1
```

Firebaseという名前で登録して、package指定時にproductに具体的に使いたいFrameworkを指定すれば良さそうです。